### PR TITLE
PLEASE DON'T MERGE - Just need this on staging

### PR DESCRIPTION
**Pull request by Nixon-Samuel**

### CODE REVIEW

File: docs/javascript/v2/guides/javascript-quickstart.mdx [hunk 1]
```diff
@@ -310,85 +244,42 @@ muteVid.addEventListener("click", () => {
...
 
-That wraps it up for this guide. We hope you had fun.
-There are plenty of cool stuff which can be done with our SDK, be sure to check the [features section](/javascript/v2/features/integration) for more information.
+
+## Next steps
 
 Here's the complete example.
 
 <Codesandbox id="5rmes" />
+
```
Comment (arjunattam) [inline docs/javascript/v2/guides/javascript-quickstart.mdx:283]: seeing a broken image on the top left of the sandbox 

Screenshot 2022-07-02 at 4 17 59 PM

File: docs/javascript/v2/guides/javascript-quickstart.mdx [hunk 2]
```diff
@@ -310,85 +244,42 @@ muteVid.addEventListener("click", () => {
...
-```css:styles.css
-.local.peer-video {
-    transform: scaleX(-1);
-}
+// listen to the connection state
+hmsStore.subscribe(onConnection, selectIsConnectedToRoom);
 ```
 
-</Tab>
+## Test the app
```
Comment (arjunattam) [inline docs/javascript/v2/guides/javascript-quickstart.mdx:275]: this section seems incomplete?

Comment (arjunattam) [reply docs/javascript/v2/guides/javascript-quickstart.mdx:275]: maybe this is where ParcelJS will be used? not seeing any mention of it in this page besides installation

File: docs/javascript/v2/guides/javascript-quickstart.mdx [hunk 3]
```diff
@@ -3,33 +3,51 @@ title: JavaScript Quickstart Guide
...
+Hello there! In this guide, we'll walk you through step-by-step instructions to create a  video conferencing application using the 100ms JavaScript SDK and test it locally on your device. In case you plan to use React, do check our [React quickstart](https://www.100ms.live/docs/javascript/v2/guides/react-quickstart) as well.
 
 TL;DR - You can find the link to the complete example over [here](https://codesandbox.io/s/hms-js-quickstart-5rmes).
 
-### Prerequisites
+## Setup a local Javascript app
+In this walkthrough, we’ll use ParcelJS a web application bundler that requires zero configuration. If you wish to use any other library or framework, you can set up your app using that. 
 
-To get started you must be familiar with rudimentary JavaScript and have [nodejs](https://nodejs.org/) installed on your machine.
+### Initialize a project
```
Comment (arjunattam) [inline docs/javascript/v2/guides/javascript-quickstart.mdx:15]: this is missing steps for installing the 100ms sdk

File: docs/javascript/v2/guides/javascript-quickstart.mdx [hunk 4]
```diff
@@ -3,33 +3,51 @@ title: JavaScript Quickstart Guide
...
+You can initialize a project using `npm init` or `yarn init`. While initializing the project you can enter the details for your project as you wish.
 
-### Installing the dependencies
+### Install Parcel
+Then, install Parcel into your app using Yarn `yarn add --dev parcel` or when using npm run `npm install --save-dev parcel`.
 
-```bash
-npm install --save @100mslive/hms-video-store@latest
+### Project setup
+Now that parcel is installed, let’s create basic source files for our video-conferencing application. Let’s create the files and folders in the below manner:
```
Comment (arjunattam) [inline docs/javascript/v2/guides/javascript-quickstart.mdx:22]: I noticed that we're using "parcel" in lower-case here. this is a common problem in our docs - there are lots of places where we use "react" instead of "React", "webrtc" instead of WebRTC. can these be caught by the linting work that we're doing?

File: docs/javascript/v2/guides/javascript-quickstart.mdx [hunk 5]
```diff
@@ -3,33 +3,51 @@ title: JavaScript Quickstart Guide
...
-### Initializing the SDK
+## Add basic functionalities
+
+### Initialize the SDK
 
 Let us start with initializing the libraries. We need an instance of `HMSStore` and `HMSActions` to get started. Copy the code snippet below and paste it into your project.
 
 ```jsx:script.js
-import { HMSReactiveStore } from '@100mslive/hms-video-store';
+import {
```
Comment (arjunattam) [inline docs/javascript/v2/guides/javascript-quickstart.mdx:38]: the filename needs to be updated here

File: docs/javascript/v2/guides/javascript-quickstart.mdx [hunk 6]
```diff
@@ -43,7 +61,22 @@ const hmsActions = hms.getHMSActions();
 -   `Peer`: A participant in the video call. You are the `local peer` while others are `remote peers`.
```
Comment (arjunattam) [inline docs/javascript/v2/guides/javascript-quickstart.mdx:61]: can we come up with a better definition for Room? the current description doesn't give me much

File: docs/javascript/v2/guides/javascript-quickstart.mdx (hunk 5 above)
Comment (arjunattam) [reply docs/javascript/v2/guides/javascript-quickstart.mdx:38]: to index.js

Comment (arjunattam) [reply docs/javascript/v2/guides/javascript-quickstart.mdx:38]: also for other snippets in the page

File: docs/javascript/v2/guides/javascript-quickstart.mdx [hunk 7]
```diff
@@ -3,33 +3,51 @@ title: JavaScript Quickstart Guide
...
-const hmsStore = hms.getStore();
-const hmsActions = hms.getHMSActions();
+// Initialize HMS Store
+const hmsManager = new HMSReactiveStore();
+hmsManager.triggerOnSubscribe();
+const hmsStore = hmsManager.getStore();
+const hmsActions = hmsManager.getHMSActions();
 ```
 
 `hmsStore` will hold the complete state of the application such as details of all the participants. We can also visualize this state at any time using the
```
Comment (arjunattam) [inline docs/javascript/v2/guides/javascript-quickstart.mdx:53]: can we link to the docs where HMSStore and Actions are detailed out? e.g. https://100ms-docs-git-dontmergejusttesting-100mslive.vercel.app/docs/api-reference/javascript/v2/interfaces/HMSActions

File: docs/javascript/v2/guides/javascript-quickstart.mdx [hunk 8]
```diff
@@ -75,80 +110,73 @@ For our application, let us collect both of these fields from the user using a f
...
-We can use "selectors" that return a slice of the state to get the data piece we're interested in.
+![Join room](/guides/join-room.png)
 
-### Subscribing to the state
+Congratulations! **you now have an audio-only conference ready to use**. If you have a friend join from another device, you can have a nice friendly chat with them.
 
-The `hmsStore` is also reactive, and we can subscribe to the state.
-What this means is that we can register a function which will get called whenever the selected state changes.
-This allows us to write [declarative](https://en.wikipedia.org/wiki/Declarative_programming) code.
+### Render Local and remote peers
```
Comment (arjunattam) [inline docs/javascript/v2/guides/javascript-quickstart.mdx:136]: inconsistent casing

File: docs/javascript/v2/guides/javascript-quickstart.mdx [hunk 9]
```diff
@@ -3,33 +3,51 @@ title: JavaScript Quickstart Guide
...
 ---
 
-## Getting started
+## Overview
 
-Hello there! In this guide, we'll build a video conferencing application using the 100ms JavaScript SDK. This guide doesn't make any assumptions related to
-the framework or library choices. In case you plan to use React, do check our React quickstart as well.
+Hello there! In this guide, we'll walk you through step-by-step instructions to create a  video conferencing application using the 100ms JavaScript SDK and test it locally on your device. In case you plan to use React, do check our [React quickstart](https://www.100ms.live/docs/javascript/v2/guides/react-quickstart) as well.
 
 TL;DR - You can find the link to the complete example over [here](https://codesandbox.io/s/hms-js-quickstart-5rmes).
```
Comment (arjunattam) [inline docs/javascript/v2/guides/javascript-quickstart.mdx:10]: using "here" or "click here" to create hyperlinks hides an opportunity to provide the user with more useful info. e.g. this can be reframed as 

```suggestion
TL;DR Try out this complete example in [CodeSandbox](https://codesandbox.io/s/hms-js-quickstart-5rmes).
```

File: docs/javascript/v2/guides/javascript-quickstart.mdx [hunk 10]
```diff
@@ -270,29 +212,21 @@ Let's add 2 buttons on the bottom of the page and call the methods on `hmsAction
...
-import {
-  selectIsLocalAudioEnabled,
-  selectIsLocalVideoEnabled,
-} from "@100mslive/hms-video-store";
-
-const muteAud = document.getElementById("mute-aud");
-const muteVid = document.getElementById("mute-vid");
-
-muteAud.addEventListener("click", () => {
+//Mute and unmute audio
```
Comment (arjunattam) [inline docs/javascript/v2/guides/javascript-quickstart.mdx:215]: spacing after `//`

File: docs/javascript/v2/guides/javascript-quickstart.mdx [hunk 11]
```diff
@@ -310,85 +244,42 @@ muteVid.addEventListener("click", () => {
...
+hmsStore.subscribe(onConnection, selectIsConnectedToRoom);
 ```
 
-</Tab>
+## Test the app
 
-That wraps it up for this guide. We hope you had fun.
-There are plenty of cool stuff which can be done with our SDK, be sure to check the [features section](/javascript/v2/features/integration) for more information.
+
+## Next steps
```
Comment (arjunattam) [inline docs/javascript/v2/guides/javascript-quickstart.mdx:278]: only the last line of this section feels apt for the title "next steps"

File: docs/javascript/v2/guides/javascript-quickstart.mdx [hunk 12]
```diff
@@ -43,7 +61,22 @@ const hmsActions = hms.getHMSActions();
...
+const peersContainer = document.getElementById("peers-container");
+const leaveBtn = document.getElementById("leave-btn");
+const muteAud = document.getElementById("mute-aud");
+const muteVid = document.getElementById("mute-vid");
+const controls = document.getElementById("controls");
+```
+
+### Join/leave a room
 
 To join a room (a video call), we need to call the `join` method on `hmsActions` and it requires us to pass a `config` object.
```
Comment (arjunattam) [inline docs/javascript/v2/guides/javascript-quickstart.mdx:81]: can we point to the docs for this method in the API reference?

Comment (arjunattam) [reply docs/javascript/v2/guides/javascript-quickstart.mdx:81]: we can use this quickstart as an opportunity to train the user to use the reference section

File: docs/javascript/v2/guides/javascript-quickstart.mdx (hunk 8 above)
Comment (arjunattam) [reply docs/javascript/v2/guides/javascript-quickstart.mdx:136]: also feels like this should be called "Render video tiles for peers"
